lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -709,17 +709,37 @@ def is_privilege?(user_or_attribute, operation, object_or_attribute)
       end
     end
 
+    # Returns true if the user has the operation on the object, and that operation is granted by
+    # the user attribute scope or its descendants
+    def is_privilege_in_role?(user_or_attribute, operation, object_or_attribute, user_attribute_scope)
```

Comment:
For some reason this comment is marked as hidden in Github - do you still want this addressed?

Reply:
Yes, please

Reply:
it's an excellent point, and frankly it probably should be more like `is_privilege_via_attribute` or something, since roles aren't really a PM thing anyways